server/src/main/java/io/crate/planner/operators/Eval.java

```diff
@@ -126,7 +126,15 @@ public FetchRewrite rewriteToFetch(TableStats tableStats, Collection<Symbol> use
         }
         for (Symbol output : outputs) {
             newReplacedOutputs.put(output, mapToFetchStubs.apply(output));
-            SymbolVisitors.intersection(output, newSource.outputs(), newOutputs::add);
+
```

Comment:
With `SymbolVisitors.intersection` we add to the `newOutputs` what it actually contains:

contains alias -> add alias
contains unwrapped alias -> add unwrapped

this is changed to

contains alias -> add alias
contains unwrapped alias -> add alias anyway